codebuild_specs/aggregate_e2e_reports.yml

```diff
@@ -4,7 +4,12 @@ env:
 phases:
   build:
     commands:
-      - source ./shared-scripts.sh && _waitForJobs $CODEBUILD_SOURCE_VERSION $WAIT_FOR_IDS
+      - cd ./scripts
+      - yarn install
+      - npm install -g ts-node
+      - npm install aws-sdk
+      - ts-node ./wait-for-all-codebuild.ts $CODEBUILD_RESOLVED_SOURCE_VERSION ../$WAIT_FOR_IDS_FILE_PATH AmplifyCLI-E2E-Testing
```

Comment:
The bash "wait for all" script was getting too complicated. I changed it to a TS script here.